### PR TITLE
made ourstory more responsive

### PR DESCRIPTION
**Pull request by RachellCalhoun**
fixed size of images, deleted fixed height/width of text. added push/pull so images will come first in mobile view.

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
@jinpark Here's another small fix. can you check it out and let me know if it LGTY

File: content/templates/content/our_story.html [hunk 1]
```diff
@@ -8,11 +8,11 @@
         {% if forloop.first %}
         <h1 class="our-story-title text-center text-uppercase">Our Story</h1>
         {% endif %}
-        <div class="col-md-6">
-            <h3 class="center-block story-text">{{ story.text | safe }}</h3>
+        <div class="col-md-6 col-md-push-6">
+            {%if story.img %}<img class="center-block img-responsive our-story-img"  src="{{ story.img.url }}" alt="Generic placeholder image" width="200" height="200">{% endif %}
```
Comment (jinpark) [inline content/templates/content/our_story.html]: would this be the same if you set the width and height in a class?

**Comment by RachellCalhoun** [comment]
@jinpark better?

File: content/static/css/content.css [hunk 2]
```diff
@@ -87,8 +87,10 @@ body {
 }
 
 .our-story-img {
-  max-height: 350px;
-  max-width: 70%;
+  max-height: 250px;
```
Comment (jinpark) [inline content/static/css/content.css:90]: is this what you want? `max-height:250px` is not the same as `height=200`

**Comment by RachellCalhoun** [comment]
yeah it was too small.

**Comment by jinpark** [comment]
lgtm then :)

**Pull request merged**